src/containers/orders/order/order-table/tests/order-table.test.js

```diff
@@ -1,35 +1,164 @@
 import React from 'react';
+import { render, screen, act, fireEvent } from '@testing-library/react';
 import { useDispatch, useSelector } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
 import OrderTable from '../order-table';
+import Modal from '../../../../../components/modal/modal';
+import {
+  mockQueryData,
+  mockQueryDataConstructor
+} from '../../../cart/cart-item/tests/cart-item.variables';
 
 jest.mock('../order-table.styles', () => ({
   useStyles: () => ({})
 }));
+jest.mock('../../../../../components/modal/modal.styles', () => ({ useStyles: () => ({}) }));
+jest.mock('../../../cart/cart-item/cart-item.styles', () => ({ useStyles: () => ({}) }));
 jest.mock('react-redux');
-
-const mockCartOperations = { removeFromCart: jest.fn() };
+jest.mock('@apollo/client', () => ({
+  ...jest.requireActual('@apollo/client'),
+  useLazyQuery: () => [
+    jest.fn(),
+    {
+      loading: false,
+      error: null,
+      data: { getProductById: mockQueryData, getConstructorById: mockQueryDataConstructor }
+    }
+  ]
+}));
 
 const dispatch = jest.fn();
 useDispatch.mockImplementation(() => dispatch);
+
 const testUseSelector = (lang) => {
   useSelector.mockImplementation(() => ({
-    language: lang
+    language: lang,
+    currency: 0
   }));
 };
-let wrapper;
+
+jest.mock('react-redux');
+jest.mock('react-i18next', () => ({
+  useTranslation: () => ({
+    t: (key) => key,
+    i18n: { changeLanguage: jest.fn() }
+  })
+}));
 
 const props = {
-  currency: 0,
-  calcPrice: 100,
-  cartLoading: false,
+  cartLoading: true,
   cartQuantityLoading: false,
-  items: [{ product: { product_id: 'some id' }, price: 10 }],
+  items: [
+    {
+      product: { product_id: 'some id' },
+      sizeAndPrice: {
+        price: [
+          {
+            value: 1000,
+            currency: 'UAH'
+          },
+          {
+            value: 37,
+            currency: 'USD'
+          }
+        ],
+        size: { _id: 'test', name: 'S' }
+      }
+    }
+  ],
```

Comment:
Maybe we should remove huge constants to test.variables.js and import them here.

Reply:
hi. i removed some constants. which i dont use

Reply:
if i import a constant the test can't reach them